primitives/src/channel.rs

```diff
@@ -51,7 +55,7 @@ pub struct ChannelSpec {
     )]
     pub active_from: Option<DateTime<Utc>>,
     /// A random number to ensure the campaignSpec hash is unique
-    pub nonce: BigNum,
+    pub nonce: Option<BigNum>,
```

Comment:
Isn't this also required? I see that in the js impl there is no `required()`, but just to check as we don't have this documented.

Reply:
Not at all